### DISCUSSION TITLE
allow setting org server side in js sdk

### DISCUSSION THREAD

**Discussion by mattkauffman23** [clerk/javascript#3651] [Feature Requests]
Why is there a limitation that a user's active org cannot be set server side? What should be a fairly simple server side process like (in pseudo):

```
const user = await clerk.createUser(...);
const org = await clerk.createOrg(...);
user.setActiveOrg(org);
```

Instead involves setting up middleware to detect an non active org state on a user, a redirect to a page that sets the org client side, and then a redirect back to the page that required an active org. That's a pretty complex and convoluted setup for what appears at least on the surface to be a simple operation and a common requirement. 

Can you add a way to the server side SDK to set a user's active org?

**Comment by jescalan** [comment]
We don't support this at the moment but it is on our roadmap! Feel free to drop an upvote or some feedback here: https://feedback.clerk.com/roadmap/39f0e922-2986-4c13-ae3f-6dc3faa5f24b
